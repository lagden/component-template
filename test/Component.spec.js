import {cleanup, render} from '@testing-library/svelte'
import Component from '../src/Component.svelte'

describe('Component', () => {
	afterEach(cleanup)

	test('should match snapshot', () => {
		const {container} = render(Component)
		expect(container).toMatchSnapshot()
	})
})
