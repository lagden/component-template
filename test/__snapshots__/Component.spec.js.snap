// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should match snapshot 1`] = `
<body>
  <div />
</body>
`;
